docs/architecture/baking.md

```diff
@@ -27,25 +27,62 @@ It packages pending operations into a block, signs it, and distributes it to the
 
 ## Becoming a baker
 
-To become a baker, you must create an account to act as a _delegate_, which is an account that is authorized to bake blocks and attest blocks that other accounts bake.
+To become a baker, you must create an account to act as a _delegate_, which is an account that is authorized to bake blocks and attest blocks that other accounts bake, and also to receive delegations from other accounts (as explained later).
 Delegates temporarily stake tez as a security deposit to ensure that they are acting according to the rules of the protocol.
-This stake stays locked for a certain number of cycles before the delegate can begin baking.
+There is a delay of a certain number of cycles from the moment when the tez are staked until the delegate can begin baking.
 Similarly, when bakers stop baking, their stake is unlocked after a certain number of cycles.
```

Comment:
This is a stale description from the old system. In the new Staking mechanism tez doesn't unlock on its own. The baker has to:
- unstake everything
- finalize stakes after 4 cycles.

Reply:
Good spot. I added a mention of unstaking and the relevant delay.